docs/views/cookies.html

```diff
@@ -54,34 +53,19 @@ <h2 class="govuk-heading-l">Our cookie message</h2>
 			  rows: [
 			    [
             { text: "seen_cookie_message" },
-            { text: "Saves a message to let us know that you have seen our cookie message" },
+            { text: "Saves a message to let us know that you've seen our cookie message" },
             { text: "28 days" }
 			    ]
 			  ]
 			}) }}
 
-      <h2 class="govuk-heading-l">Measuring website usage with Google Analytics</h2>
-
-      <p>We use Google Analytics software to collect information about how you use the GOV.UK Prototype Kit. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.</p>
-
-      <p>Google Analytics stores information about:</p>
-
-      <ul class="govuk-list govuk-list--bullet">
-        <li>the pages you visit</li>
-        <li>how long you spend on each page</li>
-        <li>how you got to the site</li>
-        <li>what you click on while you’re visiting the site.</li>
-      </ul>
-
-      <p>We don’t collect or store your personal information, so this data can’t be used to identify who you are. For more information visit our <a href="/docs/privacy-policy">privacy policy</a> page.</p>
-
-      <p>We don’t allow Google to use or share our analytics data.</p>
+			<h2 class="govuk-heading-l">Storing users' answers within prototypes</h2>
 
-      <p>Google Analytics sets the following cookies:</p>
+      <p>To help prototypes work as well as possible, each prototype stores a cookie to remember what users select on question pages. This allows prototypes to have branching pages and show users' answers on pages like the 'Check your answers' summary page.</p>
 
 			{{ govukTable({
 				classes: "app-table--fixed",
-			  caption: "Dates and amounts",
+			  caption: "Storing users' answers",
```

Comment:
Likewise, I think this could probably be more helpful so that users can understand what the table is actually listing?

Reply:
Also fixed in https://github.com/alphagov/govuk-prototype-kit/pull/845/commits/63b740469df7ad1942edd594dfbda070ae8f2825